core/templates.yml

```diff
@@ -139,6 +139,11 @@ keys:
         default: ma
         alias: extension
 
+    # needed by the Alias import as reference functionality
+    alias.extension:
```

Comment:
Is extension the right term here? Other *.extension in this file are reserved for file extension types. This doesn't seem like it. What does this alias.extension actually evaluate to?

Reply:
The functionality we're adding allow the user to import a 3D file as reference in Alias.
In order to do this, we have to convert the file to an Alias reference file (a .wref file)
We're using a template to define the path where this reference file will be stored. To avoid losing the source file type, we are also adding its extension to the file path, using this {alias.extension} key
For example, if the path to the file we want to import as reference is
/path/to/the/file.jt
once converted to an Alias reference file, its path will be
/path/to/the/file_jt.wref 
with the following template definition
@root_path/{name}_{alias.extension}.wref
where the value of {alias.extension} is the source path file extension
So, for me, this is definitely a file extension type but maybe we can use another key name :)

Reply:
Ah, got it! Thank you!